### PR TITLE
README.md: add all runtime dependencies

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca <runcom@redhat.com>

**- What I did**

**- How I did it**

**- How to verify it**

**- Description for the changelog**

### CODE REVIEW

**Comment by runcom** [comment]
@rhatdan @mrunalp PTAL and merge (w/o /test all of course)

File: README.md [hunk 1]
```diff
@@ -77,7 +77,12 @@ For sync communication we have an IRC channel #CRI-O, on chat.freenode.net, that
 
 ## Getting started
 
-### Prerequisites
+### Runtime dependencies
+
+- runc
```
Comment (mrunalp) [inline README.md]: Make it `runc or cc-runtime or any OCI compatible runtime`

Comment (runcom) [reply README.md]: awesome yeah

Comment (runcom) [reply README.md]: done!

**Comment by rhatdan** [comment]
LGTM

**Comment by runcom** [comment]
> Did this go into or does this change need to be in the 1.8 or 1.0 branches?

this is a redme fix, we're not back porting it anywhere

**Pull request merged**